astroquery/mast/observations.py

```diff
@@ -710,6 +710,9 @@ def download_products(self, products, download_dir=None,
         response : `~astropy.table.Table`
             The manifest of files downloaded, or status of files on disk if curl option chosen.
         """
+        # If the products list is a row we need to cast it as a table
+        if type(products) == Row:
```

Comment:
Is there a reason for doing `type()` rather than `isinstance()`? I see that this is following the rest of the code, so I would not hold up the merging on it, but unless @ceb8 points out a reason for this preference, I would suggest short cleanup, follow-up PR.

Reply:
@bsipocz The reason is that when I originally wrote that code I didn't know about `isinstance()`. 😝 We'll add this cleanup to the next PR.

Reply:
@bsipocz The reason is that when I originally wrote that code I didn't know about `isinstance()`. 😝 We'll add this cleanup to the next PR.